packages/backend/src/peripherals/loopring/LoopringClient.ts

```diff
@@ -4,10 +4,15 @@ import {
   Logger,
   RateLimiter,
 } from '@l2beat/common'
+import { UnixTime } from '@l2beat/types'
 
-import { LoopringResponse } from './schemas'
+import { LoopringBlock } from './schemas'
 
-type Block = number | 'finalized'
+interface Block {
```

Comment:
This type is returned from LoopringClient but is not exported and has a generic name. Maybe it should be named LoopringBlock and the schema renamed to LoopringBlockResponse